elements/rh-band/src/rh-band.scss

```diff
@@ -93,14 +103,13 @@ $aside-lg: 300px;
 
     // Remove background color for print
     @include rh-no-print-background;
+
+
   }
   &__container {
-    padding-top:    0;
```

Comment:
Why move the padding off the container?

Reply:
The template is designed for the padding to go on the container element and not the wrapper. The wrapper is for the background and image settings (overlays, etc.)

Reply:
+1 and I love that table!  That's going in the README!